packages/fxa-auth-server/lib/payments/google-play/subscriptions.ts

```diff
@@ -0,0 +1,98 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Container from 'typedi';
+import { internalValidationError } from '../../../lib/error';
+import { AppConfig } from '../../types';
+import { PlayBilling } from './play-billing';
+import {
+  AbbrevPlayPurchase,
+  GooglePlaySubscription,
+  MozillaSubscriptionTypes,
+} from 'fxa-shared/subscriptions/types';
+import { SubscriptionPurchase } from './subscription-purchase';
+import { StripeHelper } from '../stripe';
```

Comment:
Hmm I have that extension enabled, since it is recommended for FxA. I just enabled a setting for the extension called "TypeScript Hero > Imports: Organize on Save", and that seemed to do the trick. I'll do that for each modified file.